modules/scip/9.2.0/source.json

```diff
@@ -0,0 +1,11 @@
+{
+    "url": "https://github.com/scipopt/scip/archive/refs/tags/v920.tar.gz",
+    "integrity": "sha256-a22dc20f44e99bfec071889fd5af2bfc57c4af14b76d777d1312006616346f7c=",
```

Comment:
Please `bazel run //tools:calc_integrity --` with the correct arguments to update the integrity value to match the checksum of the downloaded archive.

Reply:
apart from the unreliable archive, the rest is fine.